src/message/logging.go

```diff
@@ -17,18 +17,23 @@ import (
 type LogLevel int
 
 const (
-	// WarnLevel level. Non-critical entries that deserve eyes.
-	WarnLevel LogLevel = iota
+	// Supported log levels. These are in order of increasing severity, and
+	// match the constants in the log/slog package.
+
+	// DebugLevel level. Usually only enabled when debugging. Very verbose logging.
+	DebugLevel LogLevel = -4
 	// InfoLevel level. General operational entries about what's going on inside the
 	// application.
-	InfoLevel
-	// DebugLevel level. Usually only enabled when debugging. Very verbose logging.
-	DebugLevel
+	InfoLevel LogLevel = 0
+	// WarnLevel level. Non-critical entries that deserve eyes.
+	WarnLevel LogLevel = 4
 	// TraceLevel level. Designates finer-grained informational events than the Debug.
-	TraceLevel
+	TraceLevel LogLevel = 8
```

Comment:
Trace level was meant to be a level below debug level (with more output in the logs) - Maru is a bit weird bc it was a bunch of refactored code that wound up being smushed quickly into a repo. The difference you see at Trace is due to this: https://github.com/defenseunicorns/maru-runner/blob/77f52c9d13281e0e0d9b843ccc72ad8727f6789d/src/message/message.go#L70 which should probably be refactored to use slogs levels with TraceLevel likely being a `-8` that is checked in those printers.

We should probably also add an additional errorlevel to properly handle the error messaging if that level is chosen.

Reply:
Updated!